utils/node_local_test.py

```diff
@@ -1239,8 +1260,15 @@ def log_test(conf,
              check_write=False):
     """Run the log checker on filename, logging to stdout"""
 
+    # Check if the log file has wrapped, if it has then log parsing checks do
+    # not work correctly.
+    if os.path.exists('{}.old'.format(filename)):
+        raise Exception('Log file exceeded max size')
+    fstat = os.stat(filename)
     if not quiet:
-        print('Running log_test on {}'.format(filename))
+        print('Running log_test on {} {}'.format(filename,
+                                                 sizeof_fmt(fstat.st_size)))
+    print(conf.max_log_size)
```

Comment:
Bare print here.  Was this left-over debug or should there be an explanation of what's being printed here?

Reply:
Left over.  I'll fix.